_meta/config/common.p2.yml.tmpl

```diff
@@ -32,6 +33,17 @@ inputs:
         - filesystem
         data_stream.dataset: system.filesystem
 
+  # Collecting log files
+  #- type: filestream
+  #  id: your-first-filestream-input
```

Comment:
The indentation level here doesn't match what it is in the rest of the file, particularly the `system/metrics` input above.

Reply:
Let me know if you still see the issue. I tried a second editor and so far it seems to work. It should be all spaces.

Reply:
If I remove the `#` characters manually it works. I have an editor shortcut to uncomment lines and it gets confused by the lack of leading space after the `#` in `#-type`.

Reply:
> it gets confused by the lack of leading space after the # in #-type.

Using a space for comments but not for the actual config is kind of an old convention we have always used in Beats configs. This makes it easy for the user to differentiate what is a comment and what is an actual config option (or that is the goal). That is also why I adjusted https://github.com/elastic/elastic-agent/pull/2573/files/b82af00cf2a22d0130e82362d5ad16aa647eb42b#diff-9d6103f3b09e718c26f10c21537359bb735da8a8ea18074b8b072269c17b7fb8R9 because it was not consistent.